### ISSUE TITLE
Update README.md to specify config.ini location

### ISSUE THREAD

**Issue by domupls** [aicenter/road-graph-tool#58]
Update README.md to specify which folder should the `config.ini` file be saved in - `config-EXAMPLE.ini` is in top folder (so I'd guess here) but `credentials_config.py` expects it to be in the same folder (python/roadgraphtool/):
`class CredentialsConfig:
    CONFIG_PATHS = [f'{pathlib.Path(__file__).parent}/config.ini']`

**Comment by zlochina** [comment]
`pathlib.Path(__file__).parent` takes parent of the current directory. So actually it expects `config.ini` in the directory, which one hierarchal level above

**Comment by XrayT3** [comment]
Yes, good point. I'm going to change it with my next pull-request.

**Comment by domupls** [comment]
I'd suggest in the top directory or in a completely separate directory from the scripts and such. 
I personally already use it for some bash and python scripts that are scattered across the project, residing both in the top-level directory and various subdirectories.

Maybe some discussion is in order to decide what's the best way to handle/store the configuration file.

**Comment by zlochina** [comment]
@F-I-D-O, I agree with this suggestion to put `config.ini` in the root directory of the project. Do you approve of this relocation? And also what other scripts beside `credentials_config.py` are dependent on current location of the config file?  

> I'd suggest in the top directory or in a completely separate directory from the scripts and such. I personally already use it for some bash and python scripts that are scattered across the project, residing both in the top-level directory and various subdirectories.
> 
> Maybe some discussion is in order to decide what's the best way to handle/store the configuration file.

**Comment by F-I-D-O** [comment]
I think we can put the config to the root. I see no problem with that, as we do not plan to distribute the project as a Python package. 

Considering the places where the location is hardcoded, I do not know about anything besides credentials_config.py.
